server/src/main/java/org/opensearch/indices/IndicesRequestCache.java

```diff
@@ -756,13 +756,6 @@ long count() {
         return cache.count();
     }
 
-    /**
-     * Returns the current size in bytes of the cache
-     */
-    long getSizeInBytes() {
-        return cache.stats().getTotalSizeInBytes();
-    }
-
```

Comment:
Why are we removing this? I think it should still be needed until unless we fully rely on new stats.
I see you are relying on index specific stats in tests but gives us no way to get overall node stats(if needed). Lets add this back and use the same in test for now.

Reply:
I'd removed it because if PLUGGABLE_CACHE is off, we want cache.stats().getTotalSizeInBytes() to return 0 + the API to all be 0 (because it's using the noop stats holder). So if we call this function it will incorrectly return 0. 

We could do a workaround similar to count(), where count() returns the actual count and stats().getEntries() returns 0, but that would require adding some function sizeInBytes() to the ICache interface, and I figured that'd be too much to support a temporary change like this. Let me know what you think.

Reply:
Yeah we need to remove this. I see that this was added by us and I missed the fact that it is retrieving the new cache stats (whereas I thought it is retrieving the old stats).